kn/fotos/media/fotos.js

```diff
@@ -405,16 +478,36 @@
       maxWidth -= 200;
     }
     if (width > maxWidth) {
-        height *= maxWidth/width;
-        width  *= maxWidth/width;
+      height *= maxWidth/width;
+      width  *= maxWidth/width;
     }
     if (height > maxHeight) {
-        width  *= maxHeight/height;
-        height *= maxHeight/height;
+      width  *= maxHeight/height;
+      height *= maxHeight/height;
+    }
+    // keep up to date with entities.py
```

Comment:
Tja...
Dezelfde grootte staat in feite ook al in de CSS (`870px`). Ik zou op zich wel een echte refresh kunnen doen, maar dat maakt het roteren wel een stuk langzamer. De foto moet opnieuw gecached worden op de server, dus dat kost 1-2 HTTP roundtrips en een keer `convert` aanroepen op de server. Verder zie ik nog een paar haken en ogen, maar dat zou te doen moeten zijn, al kost het waarschijnlijk wel wat meer code wil je de huidige snelheid en flexibiliteit behouden.